doc/user-manual/language/polarity.lagda.rst

```diff
@@ -0,0 +1,109 @@
+..
+  ::
+  {-# OPTIONS --polarity #-}
+
+  module language.polarity where
+
+  open import Agda.Builtin.Nat
+
+.. _polarity:
+
+======================
+ Polarity Annotations
+======================
+
+Agda now supports explicitely annotating functions arguments and datatype
+parameters with their polarities, which the positivity checker can then use to
+infer positivity. This experimental feature is hidden behind the
+:option:`--polarity` option. Here are some sample uses of the different
+annotations: ::
+
+  strictly-positive : @++ Set → Set
```

Comment:
Perhaps you could also add some examples of how these annotations are combined when two of these functions are composed?